src/gdclasses/OpenXRConfig.cpp

```diff
@@ -306,3 +309,37 @@ void OpenXRConfig::stop_passthrough() {
 		passthrough_wrapper->stop_passthrough();
 	}
 }
+
+godot::Array OpenXRConfig::get_play_space() {
+	ARVRServer *server = ARVRServer::get_singleton();
+	Array arr;
+	Vector3 sides[4] = {
+		Vector3(-0.5f, 0.0f, -0.5f),
```

Comment:
Ok this is where things do get weird. When reading the documentation ( https://www.khronos.org/registry/OpenXR/specs/1.0/html/xrspec.html#xrGetReferenceSpaceBoundsRect ) it seems like we're going to retrieve extents, it's really weird because on one hand they talk about extents from the origin point, and then they talk about width and height. 

For SteamVR this seems to definitely be returning width and height. But we could end up having a bounding rectangle that is half the required size if others runtimes interpret the specs differently, I definitely started off thinking these were extents.

Reply:
Ok, I think this is a mixup between the words `extent`, which means area and `extend` which means something that is increased in size I guess. And I wonder if that is a misunderstand in Godot because we have a few fields called extent that are indeed half sizes.

Reading up in a dictionary, extent covers the full area, so the usage of total width and height seems correct